### DISCUSSION TITLE
之前好好的，今天突然就无法运行了

### DISCUSSION THREAD

**Discussion by yan13256** [LmeSzinc/AzurLaneAutoScript#1759] [Q&A] [answered]
Repository: https://gitee.com/LmeSzinc/AzurLaneAutoScript
PypiMirror: https://pypi.tuna.tsinghua.edu.cn/simple
Language: zh-CN
Rest of the configs are the same as default

Last command: "D:/AzurLaneAutoScript/toolkit/Git/mingw64/bin/git.exe" fetch origin master
error_code: 128
Please check your deploy settings in config/deploy.yaml and re-open Alas.exe

**Comment by LmeSzinc** [comment, marked as answer]
#1755

**Comment by aowflower** [comment]
怪了，我明明10月遇到这个问题，改了后好了的，不知道为啥今天又出这个问题了。难道又挂了？
